src/shared/port-provider.js

```diff
@@ -56,7 +56,9 @@ import { isMessage, isMessageEqual, isSourceWindow } from './port-util';
  *                                                           (eg. via MessageChannel connection
  *                                                           between host and other frame)
  *
- * @implements Destroyable
+ * Immediately after the class is instantiated it starts listening for port requests.
```

Comment:
It probably would make sense to move this to the constructor docs and phrase it as eg. "Begin listening to port requests from other frames."

Reply:
👍🏼